### ISSUE TITLE
bleach_whitelist is missing `<pre>`  from the markup list

### ISSUE THREAD

**Issue by i1caro** [bartTC/django-markup#28]
```
# Tags suitable for rendering markdown
markdown_tags = [
    "h1", "h2", "h3", "h4", "h5", "h6",
    "b", "i", "strong", "em", "tt",
    "p", "br",
    "span", "div", "blockquote", "code", "hr",
    "ul", "ol", "li", "dd", "dt",
    "img",
    "a",
    "sub", "sup",
]
```
This results in:
```
from django_markup.markup import formatter 
aaa = """
    I am a text that was converted with **markdown**!
    I am a text that was converted with **markdown**!
    I am a text that was converted with **markdown**!
"""
formatter(aaa, filter_name='markdown')
```
```
&lt;pre&gt;<code>I am a text that was converted with **markdown**!
I am a text that was converted with **markdown**!
I am a text that was converted with **markdown**!
</code>&lt;/pre&gt;
```
instead of 
```
<pre><code>I am a text that was converted with **markdown**!
I am a text that was converted with **markdown**!
I am a text that was converted with **markdown**!
</code></pre>
```

**Comment by bartTC** [comment]
This is an (open) issue in bleach-whitelist. See https://github.com/yourcelf/bleach-whitelist/issues/11

I keep this issue open. If it's not fixed or clarified in a reasonable time in that dependency we can fix it ourself in dpaste. Personally I don't see a reason why `pre` is not allowed.

**Comment by bartTC** [comment]
This has been fixed in v1.5
